polkadot/node/subsystem-util/src/tests.rs

```diff
@@ -94,3 +94,63 @@ fn subset_predefined_generation_check() {
 		assert_eq!(v as usize, idx + 1);
 	}
 }
+
+// #[test]
+// fn test_availability_chunk_indices() {
```

Comment:
probably not the right place for this test

Reply:
I think the location is fine, since `ChunkIndexCacheRegistry` is defined in `subsystem-util`. I just need to completely rewrite it since the logic and interface changed since I wrote this test.